### PR TITLE
[MRG] update(GTC): implement new GTC

### PR DESCRIPTION
**Pull request by ronioeschger**
Closes parts of https://app.asana.com/0/1187035237749018/1186660118928177

### CODE REVIEW

File: site/content/german/gtc-v2020-08-19.md [hunk 1]
```diff
@@ -138,7 +138,7 @@ Zwischen Ihnen als Kunde (Sie, Ihnen, etc. oder Kunde), der Skribble E-Signing-P
 
 {{% markdown legal %}}
 ### Datenschutz
-1. Für sämtliche Informationen zum Umgang mit Kunden- und Systemnutzerdaten verweist Skribble auf seine gesonderte Datenschutzerklärung (https://www.skribble.com/de/datenschutz).
+1. Für sämtliche Informationen zum Umgang mit Kunden- und Systemnutzerdaten verweist Skribble auf seine gesonderte [Datenschutzerklärung](https://www.skribble.com/de/datenschutz).
```
Comment (Kirastina) [inline site/content/german/gtc-v2020-08-19.md:141]:
```suggestion
1. Für sämtliche Informationen zum Umgang mit Kunden- und Systemnutzerdaten verweist Skribble auf seine gesonderten [Datenschutzbestimmungen](https://www.skribble.com/de/datenschutz).
```

**Comment by Kirastina** [comment]
To merge on August 19th 2020 (after double checking)

File: site/content/german/gtc-v2020-08-19.md [hunk 2]
```diff
@@ -113,17 +113,17 @@ Zwischen Ihnen als Kunde (Sie, Ihnen, etc. oder Kunde), der Skribble E-Signing-P
...
 4. Ausserordentliche Kündigungen können nach den folgenden Regeln auf einen früheren Zeitpunkt als auf Ende einer Vertragslaufzeit erklärt werden:
-    1. auf den Zeitpunkt gemäss Ziffer 8.4., wenn der Kunde sein besonderes Kündigungsrecht im Fall von einseitig von Skribble erklärten Erhöhungen von Vergütungen ausübt.
-    2. mit sofortiger Wirkung, nach den Bestimmungen von Ziffer 12.5.;
-    3. auf den Zeitpunkt gemäss Ziffer 22.2., wenn der Kunde sein besonderes Kündigungsrecht im Fall von Anpassungen der AGB ausübt.
-    4. mit sofortiger Wirkung, wenn ein Vertrauensdiensteanbieter seine Vertragsbeziehung mit Skribble beendet, resp. seine Leistungen nicht erbringt, und Skribble dadurch die Leistungen aus dem Kundenvertrag nicht und/oder nicht vollständig erbringen kann; in diesem Fall steht Skribble alternativ die Möglichkeit zu, die Bereitstellung der Dienstleistung nach Benachrichtigung des Kunden hierüber temporär zu unterbrechen.
+    - auf den Zeitpunkt gemäss Ziffer 8.4., wenn der Kunde sein besonderes Kündigungsrecht im Fall von einseitig von Skribble erklärten Erhöhungen von Vergütungen ausübt.
```
Comment (Kirastina) [inline site/content/german/gtc-v2020-08-19.md:116]:
```suggestion
    1. auf den Zeitpunkt gemäss Ziffer 8.4., wenn der Kunde sein besonderes Kündigungsrecht im Fall von einseitig von Skribble erklärten Erhöhungen von Vergütungen ausübt.
```

File: site/content/german/gtc-v2020-08-19.md [hunk 3]
```diff
@@ -113,17 +113,17 @@ Zwischen Ihnen als Kunde (Sie, Ihnen, etc. oder Kunde), der Skribble E-Signing-P
...
+    - mit sofortiger Wirkung, nach den Bestimmungen von Ziffer 12.5.;
```
Comment (Kirastina) [inline site/content/german/gtc-v2020-08-19.md:117]:
```suggestion
    2. mit sofortiger Wirkung, nach den Bestimmungen von Ziffer 12.5.;
```

File: site/content/german/gtc-v2020-08-19.md [hunk 4]
```diff
@@ -113,17 +113,17 @@ Zwischen Ihnen als Kunde (Sie, Ihnen, etc. oder Kunde), der Skribble E-Signing-P
...
+    - auf den Zeitpunkt gemäss Ziffer 22.2., wenn der Kunde sein besonderes Kündigungsrecht im Fall von Anpassungen der AGB ausübt.
```
Comment (Kirastina) [inline site/content/german/gtc-v2020-08-19.md:118]:
```suggestion
    3. auf den Zeitpunkt gemäss Ziffer 22.2., wenn der Kunde sein besonderes Kündigungsrecht im Fall von Anpassungen der AGB ausübt.
```

File: site/content/german/gtc-v2020-08-19.md [hunk 5]
```diff
@@ -113,17 +113,17 @@ Zwischen Ihnen als Kunde (Sie, Ihnen, etc. oder Kunde), der Skribble E-Signing-P
...
+    - mit sofortiger Wirkung, wenn ein Vertrauensdiensteanbieter seine Vertragsbeziehung mit Skribble beendet, resp. seine Leistungen nicht erbringt, und Skribble dadurch die Leistungen aus dem Kundenvertrag nicht und/oder nicht vollständig erbringen kann; in diesem Fall steht Skribble alternativ die Möglichkeit zu, die Bereitstellung der Dienstleistung nach Benachrichtigung des Kunden hierüber temporär zu unterbrechen.
```
Comment (Kirastina) [inline site/content/german/gtc-v2020-08-19.md:119]:
```suggestion
    4. mit sofortiger Wirkung, wenn ein Vertrauensdiensteanbieter seine Vertragsbeziehung mit Skribble beendet, resp. seine Leistungen nicht erbringt, und Skribble dadurch die Leistungen aus dem Kundenvertrag nicht und/oder nicht vollständig erbringen kann; in diesem Fall steht Skribble alternativ die Möglichkeit zu, die Bereitstellung der Dienstleistung nach Benachrichtigung des Kunden hierüber temporär zu unterbrechen.
```

File: site/content/german/gtc-v2020-08-19.md [hunk 6]
```diff
@@ -113,17 +113,17 @@ Zwischen Ihnen als Kunde (Sie, Ihnen, etc. oder Kunde), der Skribble E-Signing-P
...
 5. Skribble kann diesen Vertrag zu jeder Zeit mit sofortiger Wirkung kündigen oder die Bereitstellung der Dienstleistung nach Benachrichtigung des Kunden hierüber unterbrechen, falls der Kunde oder seine Systemnutzer wesentliche Bestimmungen des Kundenvertrags verletzt haben und dieser Verstoss entweder nicht behoben werden kann oder nach entsprechender Aufforderung nicht innerhalb einer Frist von 10 Tagen behoben worden ist. Wenn der Kunde mit der Bezahlung von Vergütungen in Verzug ist, gilt dies als Verletzung einer wesentlichen Bestimmung des Kundenvertrags.
 
 ### Sistierung des Vertragsverhältnisses bzw. des Zugangs zu den Skribble-Diensten
 1.  Skribble ist berechtigt, den Zugang von Systemnutzern zu Skribble-Diensten zu sistieren oder einzuschränken:
-    1. wenn der Kunde mit der Bezahlung von Vergütungen in Verzug ist;
-    2. wenn der Kunde oder ein Systemnutzer gegen den Kundenvertrag oder die Nutzungsbedingungen verstossen;
-    3. wenn aufgrund von Umständen, die im Risikobereich des Kunden liegen, der unbeeinträchtigte Betrieb von Skribble gefährdet wird.
+    - wenn der Kunde mit der Bezahlung von Vergütungen in Verzug ist;
```
Comment (Kirastina) [inline site/content/german/gtc-v2020-08-19.md:124]:
```suggestion
    1. wenn der Kunde mit der Bezahlung von Vergütungen in Verzug ist;
```

File: site/content/german/gtc-v2020-08-19.md [hunk 7]
```diff
@@ -113,17 +113,17 @@ Zwischen Ihnen als Kunde (Sie, Ihnen, etc. oder Kunde), der Skribble E-Signing-P
...
+    - wenn der Kunde oder ein Systemnutzer gegen den Kundenvertrag oder die Nutzungsbedingungen verstossen;
```
Comment (Kirastina) [inline site/content/german/gtc-v2020-08-19.md:125]:
```suggestion
    2. wenn der Kunde oder ein Systemnutzer gegen den Kundenvertrag oder die Nutzungsbedingungen verstossen;
```

File: site/content/german/gtc-v2020-08-19.md [hunk 8]
```diff
@@ -113,17 +113,17 @@ Zwischen Ihnen als Kunde (Sie, Ihnen, etc. oder Kunde), der Skribble E-Signing-P
...
+    - wenn aufgrund von Umständen, die im Risikobereich des Kunden liegen, der unbeeinträchtigte Betrieb von Skribble gefährdet wird.
```
Comment (Kirastina) [inline site/content/german/gtc-v2020-08-19.md:126]:
```suggestion
    3. wenn aufgrund von Umständen, die im Risikobereich des Kunden liegen, der unbeeinträchtigte Betrieb von Skribble gefährdet wird.
```

**Comment by Kirastina** [comment]
To merge on August 19th 2020 (after double checking)

**Comment by Kirastina** [comment]
To merge on August 19th 2020 (after double checking)

**Comment by Kirastina** [comment]
OK to MRG by Simon : https://app.asana.com/0/1148056102485097/1189022701016225/f

File: site/content/english/gtc-v2020-08-19.md [hunk 9]
```diff
@@ -0,0 +1,227 @@
+---
+title: General Terms and Conditions (GTC) of Skribble
+slug: gtc
+aliases:
+  - gtc/archive/20200819
+  - gtc-v2020-08-19
```
Comment (betatim) [inline site/content/english/gtc-v2020-08-19.md:6]: I think this alias is a legacy one which we maintain for old versions but don't need for new versions. Propose to remove it.

File: site/content/french/gtc-v2020-08-19.md [hunk 10]
```diff
@@ -0,0 +1,231 @@
+---
+title: Conditions Générales (CG) de Skribble
+slug: cg
+aliases:
+  - cg/archives/20200819
+  - cg-v2020-08-19
```
Comment (betatim) [inline site/content/french/gtc-v2020-08-19.md:6]: as above

File: site/content/german/gtc-v2020-08-19.md [hunk 11]
```diff
@@ -0,0 +1,231 @@
+---
+title: Allgemeine Geschäftsbedingungen (AGB) von Skribble
+slug: agb
+aliases:
+  - agb/archiv/20200819
+  - agb-v2020-08-19
```
Comment (betatim) [inline site/content/german/gtc-v2020-08-19.md:6]: as above

**Comment by betatim** [comment]
As in #426, merging now, legacy aliases in a new PR.

**Pull request merged**